frontend/src/component/insights/componentsChart/ProjectHealthChart/ProjectHealthChart.tsx

```diff
@@ -113,6 +113,41 @@ export const ProjectHealthChart: VFC<IProjectHealthChartProps> = ({
                     ? {}
                     : {
                           parsing: { yAxisKey: 'health', xAxisKey: 'date' },
+                          scales: {
+                              y: {
+                                  min: 0,
```

Comment:
This is main change, but to avoid other scales default options from being overridden, I need to pull it out. Other options is to create 2 new props for min and max, but that does not scale nicely.